otter/log/formatters.py

```diff
@@ -242,35 +281,14 @@ def Observer(eventDict):
             "otter_facility": eventDict.get("system", "otter"),
         }
 
-        if eventDict.get("isError", False):
-            level = 3
-
-            if 'failure' in eventDict:
-                message = repr(eventDict['failure'].value)
-                log_params['traceback'] = eventDict['failure'].getTraceback()
-
-            if 'why' in eventDict and eventDict['why']:
-                message = '{0}: {1}'.format(eventDict['why'], message)
-
-        else:
-            level = 6
-
-        if not message:
-            message = eventDict["message"][0] if eventDict["message"] else ""
-
-        log_params.update({
-            "message": message,
-            "level": eventDict.get("level", level),
-        })
-
         if "file" in eventDict:
```

Comment:
I know this PR hasn't changed this part, but I wonder if these lines are necessary - it seems like the for loop below would handle it?